BDArmory/Damage/HitpointTracker.cs

```diff
@@ -1467,18 +1460,29 @@ void SetHullMass()
                     HullTypeNum = HullInfo.materials.FindIndex(t => t.name == "Aluminium") + 1;
                     part.maxTemp = part.partInfo.partPrefab.maxTemp;
                 }
+
                 hullInfo = HullInfo.materials[HullInfo.materialNames[(int)HullTypeNum - 1]];
+            }
```

Comment:
This misplaced curlybrace was the reason hull materials weren't applying in flight